kafka-to-log/README.md

```diff
@@ -7,12 +7,12 @@ This example shows how to read a Kafka topic, using the camel `kafka` component,
 2. A topic named `test-topic` with some messages in it
 
 ## How to run
-1. Open the `kafka.camel.yaml` file and set the `brokers`, `topic` and `saslJaasConfig` values to match your Kafka cluster
+1. Open the `kafka-to-log.camel` file and set the `brokers`, `topic` and `saslJaasConfig` values to match your Kafka cluster
```

Comment:
.yaml is missing?

Reply:
Right, my bad, fixed now, thanks for catching this up